osf/models/mixins.py

```diff
@@ -1593,7 +1593,8 @@ def _get_spam_content(self, saved_fields):
         spam_fields = self.get_spam_fields(saved_fields)
         content = []
         for field in spam_fields:
-            content.append((getattr(self, field, None) or '').encode('utf-8'))
+            values = list(self.__class__.objects.filter(id=self.id).values_list(field, flat=True))
```

Comment:
We need to make sure that none of the values in this array are a None value, or otherwise we won't be able to join them.  Can you give them an alternative value of an empty string?

Reply:
Fix it so now every NoneType will just return an empty list that is ignored if returned. Sorry this is a bit weird.